src/controller/python/chip/clusters/attribute.cpp

```diff
@@ -455,6 +453,7 @@ void pychip_ReadClient_Abort(ReadClient * apReadClient, ReadClientCallback * apC
     VerifyOrDie(apReadClient != nullptr);
     VerifyOrDie(apCallback != nullptr);
 
+    delete apReadClient;
```

Comment:
This can lead to double-free if Abort is ever called after OnDone. Was this validated? This behavior existed before, however.

Reply:
It will not be double-free. The subscription will only execute `OnDone` when an error occurs. When calling `pychip_ReadClient_Abort`, it has been confirmed whether an error occurred.